bcda/api/requests.go

```diff
@@ -117,30 +117,12 @@ func (h *Handler) BulkPatientRequest(w http.ResponseWriter, r *http.Request) {
 	h.bulkRequest(w, r, reqType)
 }
 
-func (h *Handler) BulkALRPatientRequest(w http.ResponseWriter, r *http.Request) {
-	reqType := service.DefaultRequest // historical data for new beneficiaries will not be retrieved (this capability is only available with /Group)
+func (h *Handler) ALRRequest(w http.ResponseWriter, r *http.Request) {
+	reqType := service.DefaultRequest
```

Comment:
So we recently found out the RequestType is only related to BFD. We don't have to do anything in this ticket, but future ticket will follow in removing this for ALR.